templates/web/base/waste/bulky/intro.html

```diff
@@ -0,0 +1,26 @@
+[% SET title = form.title ~%]
```

Comment:
The Claims form introduced an intro_template variable in order to provide a static block above a form on a particular page - perhaps worth reusing/generalising that into the waste template as well, and then that could be used here? Just an idea.

Reply:
Oh that's nicer isn't it. Have reused that here as you suggest.